practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/model/PageableResult.java

```diff
@@ -0,0 +1,87 @@
+package uk.ac.ox.ndph.mts.practitioner_service.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.Collection;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+/**
+ * Holder for pageable result from the usual pageable API
+ * @param <T>  type of entity in page
+ */
+public class PageableResult<T> implements Iterable<T> {
```

Comment:
This isn't used and I don't think we'd in the future. Can you remove it?

Reply:
Done